core/pom.xml

```diff
@@ -46,10 +46,6 @@
             </excludes>
           </artifactSet>
           <relocations>
-            <relocation>
-              <pattern>org.reflections.</pattern>
```

Comment:
we should remove this dependency from shading. There are some usages of `org.reflections` in other poms.

Reply:
it was used only in example/server, removed